src/org/gk/database/EventCentricViewPane.java

```diff
@@ -522,10 +522,9 @@ public void rebuildTree() {
 	
 	/**
 	 * Call this method if an Instance's type (i.e. class) is changed.
-	 * @param oldCls the old SchemaClass.
 	 * @param instance the instance whose SchemaClass has been changed.
 	 */
-	public void switchedType(GKSchemaClass oldCls, GKInstance instance) {
+	public void switchedType(GKInstance instance) {
```

Comment:
Could I keep the original method signature, just for conforming to the same signatures in other views' switchedType(). See line 387 in class GKCuratorFrame.

Reply:
Sure, if you prefer we can keep the original method signature but I had noticed that the switchedType method here and other calls around line 387 in the GKCuratorFrame class do not use the `oldCls` parameter.  I'm just concerned it is misleading to include it as a parameter.

Reply:
I guess it will not. Thanks!

Reply:
Okay, so I should keep the `oldCls` parameter in these methods and their callers?

Reply:
Sure -- the reversions to restore `oldCls` have been committed .